api/tests/src/services/users/test_create_api_key.py

```
@@ -12,7 +12,8 @@
 from tests.src.db.models.factories import UserApiKeyFactory, UserFactory
 
 
-def test_create_api_key_success(enable_factory_create, db_session: db.Session):
+@patch("src.services.users.create_api_key.import_api_key")
```

What is this patch doing? With the logic defaulting to the mock with the local AWS stuff, does it not work?

---

You're right, I think this is an artifact from an earlier iteration of this work that I forgot to remove since it does nothing. I have gone ahead and removed it.